### DISCUSSION TITLE
I need math ability

### DISCUSSION THREAD

**Discussion by bountyhunter01** [community/community#106368] [Programming Help]
### Body

I am learning programming and I wanna game programmer but my math ability is low level how get a high math ability that needs programming? I wanna recommend and give feedback!

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by gil-son** [comment]
Hello,

Organize your time, maintain your discipline and remain patient:

- https://github.com/TalalAlrawajfeh/mathematics-roadmap?tab=readme-ov-file
- https://github.com/ossu/math?tab=readme-ov-file#summary
- https://www.youtube.com/watch?app=desktop&v=pTnEG_WGd2Q

**Reply by bountyhunter01** [reply]
I really appreciate your help sir!

**Comment by ghost** [comment]
See most programmers won't need to know huge concepts of maths.Just add,subtract,divide and multiplication would do it. It also depend on what field you are choosing. If going for application development then the above mentioned maths is enough. But choosing fields like data scientist, data analyst, etc. Would require statistics and other maths concepts. So think about it

**Reply by bountyhunter01** [reply]
Thank you, sir!

**Reply by ghost** [reply]
Don't call me sir its just yuhesh

**Reply by ghost** [reply]
And Happy Maths Journey ❤️

**Reply by bountyhunter01** [reply]
Thank you yuhesh!
